samples/core/enumopencl/main.cpp

```diff
@@ -67,6 +67,9 @@ static cl_int PrintDeviceInfoSummary(const std::vector<cl::Device> devices)
 
 int main(int argc, char** argv)
```

Comment:
```suggestion
int main(int, char**)
```

An alternative way to eliminate the unused warning, drop the unused names.

Do not take both of my suggestions, just one, either is okay with me.

Reply:
Done as suggested.